### PR TITLE
[AKS] Fix AKS PR check-in pipeline

### PR DESCRIPTION
**Pull request by FumingZhang**
* fix trigger branch (from master to main)
* update pipeline agent/pool to 'ubuntu-latest'
* update scripts

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml [hunk 1]
```diff
@@ -5,15 +5,15 @@ trigger: none
...
+    - main
   paths:
     include:
     - src/aks-preview/
 
 jobs:
 - job: LiveTest
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```
Comment (andyliuliming) [inline src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml:16]: I don't think this is a good idea. we should always stick to one version.

**Review by andyliuliming — CHANGES_REQUESTED** [review]

File: src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml (hunk 1 above)
Comment (FumingZhang) [reply src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml]: Fixed in commit [4f71818](https://github.com/Azure/azure-cli-extensions/pull/3532/commits/4f71818df70f3a4e3a1aaad4cac70b541f700738).

**Review by andyliuliming — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Comment by zhoxing-ms** [comment]
@FumingZhang  May I ask if this PR title should appear in release notes as a history?

**Comment by FumingZhang** [comment]
> @FumingZhang May I ask if this PR title need to appear in release notes as a history?

No, I have changed the title.

**Pull request merged**